contracts/test/foundry/FleekERC721/GetToken.t.sol

```diff
@@ -49,25 +58,23 @@ contract Test_FleekERC721_GetToken is Test_FleekERC721_Base {
             string memory logo,
             uint24 color
         ) = CuT.getToken(tokenId);
-        assertEq(name, "New App Name");
-        assertEq(description, "New description for the app.");
-        assertEq(externalURL, "https://new-url.com");
-        assertEq(logo, TestConstants.LOGO_1);
-        assertEq(color, 0x654321);
-        assertEq(ENS, "new-ens.eth");
+        assertEq(name, newAppName);
+        assertEq(description, newDescription);
+        assertEq(externalURL, newExternalURL);
+        assertEq(logo, newLogo);
+        assertEq(color, newColor);
+        assertEq(ENS, newENS);
         assertEq(currentBuild, 1);
     }
 
-    function test_getTokenForDifferentAddresses() public {
-        vm.prank(address(1));
-        CuT.getToken(tokenId);
-        vm.prank(address(2));
-        CuT.getToken(tokenId);
-        vm.prank(address(3));
+    function testFuzz_getTokenForDifferentAddresses(address account) public {
+        vm.prank(account);
         CuT.getToken(tokenId);
```

Comment:
Should we assert something here? This isn't related to new change but something I missed in a previous review.